// @clerk/themes/dist/clerk-js/src/ui/foundations/typography.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.fontStyles = exports.fonts = exports.lineHeights = exports.letterSpacings = exports.fontWeights = exports.fontSizes = void 0;
const fontWeights = Object.freeze({
    normal: 400,
    medium: 500,
    bold: 600,
});
exports.fontWeights = fontWeights;
const lineHeights = Object.freeze({
    normal: 'normal',
    none: 1,
    shortest: 1.1,
    shorter: 1.25,
    short: 1.375,
    base: 1.5,
    tall: 1.625,
    taller: 2,
});
exports.lineHeights = lineHeights;
const letterSpacings = Object.freeze({
    tighter: '-0.05em',
    tight: '-0.025em',
    normal: '0',
    wide: '0.025em',
    wider: '0.05em',
    widest: '0.1em',
});
exports.letterSpacings = letterSpacings;
const fontSizes = Object.freeze({
    '2xs': '0.6875rem',
    xs: '0.8125rem',
    sm: '0.875rem',
    md: '1rem',
    lg: '1.125rem',
    xl: '1.25rem',
    '2xl': '2rem',
});
exports.fontSizes = fontSizes;
const fontStyles = Object.freeze({
    normal: 'normal',
});
exports.fontStyles = fontStyles;
const fonts = Object.freeze({
    main: 'inherit',
    buttons: 'inherit',
});
exports.fonts = fonts;
